### PR TITLE
Updated Marko example to support Marko 4.5.x

### PR DESCRIPTION
**Pull request by bricss**
Updated Marko example to support version 4.x with sync & async compiling modes.

### CODE REVIEW

**Comment by bricss** [comment]
@Marsup Ping

**Comment by Marsup** [comment]
Why me, I'm not even the maintainer.

**Comment by bricss** [comment]
Sorry, I just saw that you was last who was active in this repo 😃

**Comment by bricss** [comment]
@jagoda Ping 📡

**Comment by wswoodruff** [comment]
@bricss Can you please resolve these conflicts?

**Comment by wswoodruff** [comment]
@bricss I just updated the marko example to hapi v17 -- it isn't working yet though! I figured you can add your changes to it if you'd like, since there's an issue (#126) for it I'd like to know a ballpark of when you think you could get the updates in, thanks!

**Comment by wswoodruff** [comment]
Got the example working, `vision` gives marko templates the context in a `data` variable

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (wswoodruff) [inline .editorconfig]: I'm not sure if we want to introduce `.editorconfig` just yet,
I pinged the issue here just now: https://github.com/hapijs/contrib/issues/38

Also we'd want to add this to it:
```
[*.{json,yml}]
indent_size = 2
```

Comment (wswoodruff) [reply .editorconfig]: I'm sure there could be more things that could be added when reviewing the eslint hapi config, I haven't gone over it yet

Comment (bricss) [reply .editorconfig]: I can remove `.editorconfig` then

Comment (wswoodruff) [reply .editorconfig]: 👌 But I would be interested in reopening that discussion!

Comment (bricss) [reply .editorconfig]: Seems to be `done`

**Comment by wswoodruff** [comment]
Looks great, thanks!

**Comment by Marsup** [comment]
What's with all that coveralls stuff ? We don't need that, our coverage is 100%, always.

**Comment by AdriVanHoudt** [comment]
The tests will literary fail when coverage drops below 100 due to `-t 100`

**Comment by wswoodruff** [comment]
You guys want me to take it off?

**Comment by Marsup** [comment]
Would rather, yes. If you find a way to put a static badge I don't mind.

**Comment by wswoodruff** [comment]
@marsup Just turned off PR alerts for `coveralls` -- including the comments you see here; the badge on the README should still update correctly.

**Comment by wswoodruff** [comment]
@csakis we've got an example here: https://github.com/hapijs/vision/tree/master/examples/marko

**Comment by csakis** [comment]
Thanks, Williams. Five minutes after my post, I solved the problem.

**Pull request merged**